File: DAY 2/PYTHON/LEVEL 2/largestinthreenumbers.py
num = int(input("enter number: "));
numtwo = int(input("enter number: "));
numthree = int(input("enter number: "));
 
largest = num

if(largest < numtwo):
    largest = numtwo
elif(largest < numthree):
    largest = numthree
print(f"the largest is {largest}")

File: DAY 2/JAVA SCRIPT/LEVEL 2/largestinthreenumbers.js
const prompt = require("prompt-sync")();

let num = Number(prompt("Enter first number: "));
let numtwo = Number(prompt("Enter second number: "));
let numthree = Number(prompt("Enter second number: "));

let largest = num;

if (numtwo > largest) {
    largest = numtwo;
} else if (numthree > largest){
    largest = numthree;
}

console.log("The largest is", largest);
